### PR TITLE
Add Holybro kakuteH7mini

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

**Review by andyp1per — APPROVED** [review]

File: common/source/docs/common-holybro-kakuteh7mini.rst [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+-  **Processor**
+
+   -  STM32H743 32-bit processor
+   -  AT7456E Video processor for OSD
+   -  16MB data flash for logging
+
+-  **Sensors**
+
+   -  MPU6000 (accel & gyro)
+   -  BMP280 barometers
```
Comment (nkruzan) [inline common/source/docs/common-holybro-kakuteh7mini.rst:24]:
```suggestion
   -  BMP280 barometer
```
docs only suggest single baro

File: common/source/docs/common-holybro-kakuteh7mini.rst [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+- CRSF also requires a T6 connection, in addition to R6, and automatically provides telemetry. Set :ref:`SERIAL6_OPTIONS<SERIAL6_OPTIONS>` to "0".
+
+- SRXL2 requires a connection to T6 and automatically provides telemetry.  Set :ref:`SERIAL6_OPTIONS<SERIAL6_OPTIONS>` to "4".
+
+Any UART can be used for RC system connections in ArduPilot also, and is compatible with all protocols except PPM. See :ref:`common-rc-systems` for details.
+
+OSD Support
+===========
+
+The KakuteF7 Mini supports OSD using :ref:`OSD_TYPE<OSD_TYPE>` 1 (MAX7456 driver).
```
Comment (nkruzan) [inline common/source/docs/common-holybro-kakuteh7mini.rst:85]:
```suggestion
The KakuteH7 Mini supports OSD using :ref:`OSD_TYPE<OSD_TYPE>` 1 (MAX7456 driver).
```

File: common/source/docs/common-holybro-kakuteh7mini.rst [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+PWM Output
+==========
+
+TThe KakuteH7 Mini supports up to 9 outputs. M1 to M4 are accessed via the connector and are :ref:`bi-directional<bidir-dshot>` capable. M5-M8 and LED are the pads shown on the above diagram. M1-M8 and LED outputs support DShot as well as all PWM types.
```
Comment (nkruzan) [inline common/source/docs/common-holybro-kakuteh7mini.rst:90]:
```suggestion
The KakuteH7 Mini supports up to 9 outputs. M1 to M4 are accessed via the connector and are :ref:`bi-directional<bidir-dshot>` capable. M5-M8 and LED are the pads shown on the above diagram. M1-M8 and LED outputs support DShot as well as all PWM types.
```

**Review by nkruzan — CHANGES_REQUESTED** [review]
some nitpicks, as always ;)

**Comment by Hwurzburg** [comment]
thanks...fixed all

**Review by nkruzan — APPROVED** [review]

**Pull request merged**